.env

```diff
@@ -0,0 +1 @@
+DB_ELEPHANT=postgres://apowlauz:n7YdwjVfTkI0tLE1EVAd4RN2rq6zxnze@nutty-custard-apple.db.elephantsql.com:5432/apowlauz
```

Comment:
you can't push your database url to github. use your environment variables @Fiyiin

Reply:
Done